src/containers/current-item.js

```diff
@@ -1,35 +1,39 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-class CurrentItem extends React.Component{
-    render(){
-        console.log("test")
-        if(!this.props.selectedVideo && this.props.videoList){
-            return <div>waiting for video selection</div>;
-	    }
+class CurrentItem extends React.Component {
+  render() {
+    if (this.props.videoList.length == 0) {
```

Comment:
this one can fail you should do something like this:

if (videoList && videoList.length) {